proxies/events-graphql-proxy/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "events-graphql-proxy",
+  "description": "Events Helsinki GraphQL proxy.",
+  "version": "0.2.10",
+  "private": true,
+  "scripts": {
+    "clean": "rimraf --no-glob ./tsconfig.tsbuildinfo",
+    "build": "NODE_ENV=production webpack --progress --mode production --config config/webpack.config.js",
+    "dev": "cross-env GRAPHQL_PROXY_PORT=4100 nodemon",
```

Comment:
The dev evironment does not work:
```
❯ yarn dev
command not found: cross-env
```

I did install the deps first

Reply:
`yarn add cross-env` was needed. After that it worked.

Reply:
fixed